crates/swc_css_prefixer/src/prefixer.rs

```diff
@@ -701,6 +695,41 @@ impl VisitMut for Prefixer {
         }
     }
 
+    fn visit_mut_import_prelude(&mut self, import_prelude: &mut ImportPrelude) {
+        import_prelude.visit_mut_children_with(self);
+
+        if !self.added_declarations.is_empty() {
+            if let Some(ImportPreludeSupportsType::Declaration(declaration)) =
+                import_prelude.supports.take()
```

Comment:
This will set `import_prelude.supports` regardless of the value being `Some(ImportPreludeSupportsType::Declaration(...))`.

Reply:
Is this intended?

Reply:
Yes, we do it only when we have `added_declarations` not empty, it means we have more declarations in `supports()` and should convert it to `SupportConditions`